sdk/resources/Azure.ResourceManager.Resources/tests/Azure.ResourceManager.Resources.Tests.csproj

```diff
@@ -1,5 +1,16 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TestHelperProjects>Rbac1.6;</TestHelperProjects>
```

Comment:
If Rbac1.6 is not used, please change to:
```
<TestHelperProjects>;</TestHelperProjects>
```

Reply:
Done